### ISSUE TITLE
[BUG]1 GiB RAM not displayed in bashtop

### ISSUE THREAD

**Issue by Rosika2** [aristocratos/bashtop#137]
1 GiB RAM not displayed in bashtop

Hi altogether,

I´ve got a question regarding displaying the memory-value in bashtop.

Running bashtop on BodhiLinux in a virtual machine and having allocated 1 GiB (1024 MB) virtual RAM to it bashtop displays the value (or rather does not) thus:  
`Memory:         . GiB`

So basically just a dot.

Out of curiosity I temporarily allocated 2 GiB (2048 MB) virtual RAM to Bodhi. Now bashtop rather correctly displays  
`Memory:       1.97 GiB`

So is there a reason why 1 GiB RAM isn´t displayed at all?

Thanks a lot in advance.

Greetings
Rosika 

Info:
 - Bashtop version:  0.9.9+git218-850c9d0~ubuntu18.04.1
 - (Linux) Linux distribution and version: BodhiLinux 5 Release 18.04
 - Terminal used: lxterminal
 - Font used: Monospace Regular
 - Bash version, "bash --version" (version 4.4 or above is required): 4.4.18-2ubuntu1.2
 - Locales: output of "locale -v": 
```
LANG=de_DE.UTF-8
LANGUAGE=de_DE
LC_CTYPE="de_DE.UTF-8"
LC_NUMERIC=de_DE.UTF-8
LC_TIME=de_DE.UTF-8
LC_COLLATE="de_DE.UTF-8"
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES="de_DE.UTF-8"
LC_PAPER=de_DE.UTF-8
LC_NAME=de_DE.UTF-8
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=de_DE.UTF-8
LC_MEASUREMENT=de_DE.UTF-8
LC_IDENTIFICATION=de_DE.UTF-8
LC_ALL=
```

**Comment by aristocratos** [comment]
That's because I made an error in the humanizer function, will be fixed in next update. :)

**Comment by aristocratos** [comment]
@Rosika2 
Fixed in v0.9.10

**Comment by Rosika2** [comment]
@aristocratos:
Hello and thanks for your update.
I just updated my  system and got the new bashtop version.
For Linux/Bodhi it´s 
`< ii  bashtop                                0.9.9+git222-b4a3e81~ubuntu18.04.1        i386         Resource monitor that shows usage`
And indeed RAM usage is now displayed thus:
`Memory:           1.00 GiB`
That´s  phanatastic. :)

Thanks again for your work with this nice programme.

Have a nice weekend and keep safe.
Many greetings.
Rosika
